src/Data/Transformer/LoginTransformer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace App\Data\Transformer;
+
+use App\Data\Entity\Login;
+use App\Data\Transformer;
+
+class LoginTransformer extends Transformer
```

Comment:
Fractal isn't just for output formatting, it also allows you to transform incoming data. It doesn't sell itself that way, but it certainly can fulfill that role.